src/parse-css.pegjs

```diff
@@ -886,25 +886,32 @@ nl
-R  = 'r'i / '\\' '0'? '0'? '0'? '0'? [\x52\x72] ('\r\n' / [ \t\r\n\f])? / '\\r'i { return 'r'; }
+A = 'a'i / '\\' '0'? '0'? '0'? '0'? [\x41\x61] ('\r\n' / [ \t\r\n\f])? { return 'a'; }
+B = 'b'i / '\\' '0'? '0'? '0'? '0'? [\x42\x62] ('\r\n' / [ \t\r\n\f])? / '\\b'i { return 'b'; }
+C = 'c'i / '\\' '0'? '0'? '0'? '0'? [\x43\x63] ('\r\n' / [ \t\r\n\f])? { return 'c'; }
+D = 'd'i / '\\' '0'? '0'? '0'? '0'? [\x44\x64] ('\r\n' / [ \t\r\n\f])? { return 'd'; }
+E = 'e'i / '\\' '0'? '0'? '0'? '0'? [\x45\x65] ('\r\n' / [ \t\r\n\f])? { return 'e'; }
+F = 'f'i / '\\' '0'? '0'? '0'? '0'? [\x46\x66] ('\r\n' / [ \t\r\n\f])? / '\\f'i { return 'f'; }
+G = 'g'i / '\\' '0'? '0'? '0'? '0'? [\x47\x67] ('\r\n' / [ \t\r\n\f])? / '\\g'i { return 'g'; }
+H = 'h'i / '\\' '0'? '0'? '0'? '0'? [\x48\x68] ('\r\n' / [ \t\r\n\f])? / '\\h'i { return 'h'; }
+I = 'i'i / '\\' '0'? '0'? '0'? '0'? [\x49\x69] ('\r\n' / [ \t\r\n\f])? / '\\i'i { return 'i'; }
+J = 'j'i / '\\' '0'? '0'? '0'? '0'? [\x4A\x6A] ('\r\n' / [ \t\r\n\f])? / '\\j'i { return 'j'; }
+K = 'k'i / '\\' '0'? '0'? '0'? '0'? [\x4B\x6B] ('\r\n' / [ \t\r\n\f])? / '\\k'i { return 'k'; }
+L = 'l'i / '\\' '0'? '0'? '0'? '0'? [\x4C\x6C] ('\r\n' / [ \t\r\n\f])? / '\\l'i { return 'l'; }
+M = 'm'i / '\\' '0'? '0'? '0'? '0'? [\x4D\x6D] ('\r\n' / [ \t\r\n\f])? / '\\m'i { return 'm'; }
+N = 'n'i / '\\' '0'? '0'? '0'? '0'? [\x4E\x6E] ('\r\n' / [ \t\r\n\f])? / '\\n'i { return 'n'; }
+O = 'o'i / '\\' '0'? '0'? '0'? '0'? [\x4F\x6F] ('\r\n' / [ \t\r\n\f])? / '\\o'i { return 'o'; }
+P = 'p'i / '\\' '0'? '0'? '0'? '0'? [\x50\x70] ('\r\n' / [ \t\r\n\f])? / '\\p'i { return 'p'; }
+Q = 'q'i / '\\' '0'? '0'? '0'? '0'? [\x51\x71] ('\r\n' / [ \t\r\n\f])? / '\\q'i { return 'q'; }
+R = 'r'i / '\\' '0'? '0'? '0'? '0'? [\x52\x72] ('\r\n' / [ \t\r\n\f])? / '\\r'i { return 'r'; }
 S_ = 's'i / '\\' '0'? '0'? '0'? '0'? [\x53\x73] ('\r\n' / [ \t\r\n\f])? / '\\s'i { return 's'; }
-T  = 't'i / '\\' '0'? '0'? '0'? '0'? [\x54\x74] ('\r\n' / [ \t\r\n\f])? / '\\t'i { return 't'; }
-U  = 'u'i / '\\' '0'? '0'? '0'? '0'? [\x55\x75] ('\r\n' / [ \t\r\n\f])? / '\\u'i { return 'u'; }
-X  = 'x'i / '\\' '0'? '0'? '0'? '0'? [\x58\x78] ('\r\n' / [ \t\r\n\f])? / '\\x'i { return 'x'; }
-Z  = 'z'i / '\\' '0'? '0'? '0'? '0'? [\x5a\x7a] ('\r\n' / [ \t\r\n\f])? / '\\z'i { return 'z'; }
+T = 't'i / '\\' '0'? '0'? '0'? '0'? [\x54\x74] ('\r\n' / [ \t\r\n\f])? / '\\t'i { return 't'; }
+U = 'u'i / '\\' '0'? '0'? '0'? '0'? [\x55\x75] ('\r\n' / [ \t\r\n\f])? / '\\u'i { return 'u'; }
+V = 'v'i / '\\' '0'? '0'? '0'? '0'? [\x56\x76] ('\r\n' / [ \t\r\n\f])? / '\\v'i { return 'v'; }
+W = 'w'i / '\\' '0'? '0'? '0'? '0'? [\x57\x77] ('\r\n' / [ \t\r\n\f])? / '\\w'i { return 'w'; }
+X = 'x'i / '\\' '0'? '0'? '0'? '0'? [\x58\x78] ('\r\n' / [ \t\r\n\f])? / '\\x'i { return 'x'; }
+Y = 'y'i / '\\' '0'? '0'? '0'? '0'? [\x59\x79] ('\r\n' / [ \t\r\n\f])? / '\\y'i { return 'y'; }
+Z = 'z'i / '\\' '0'? '0'? '0'? '0'? [\x5A\x7A] ('\r\n' / [ \t\r\n\f])? / '\\z'i { return 'z'; }
```

Comment:
I don't think most of the additions here are used. Can remove the other changes, so it just adds `Q`?

Reply:
I can, but I think it very likely we'll need some/all of them and having it be complete now seemed likely a way to reduce errors in the future. Maybe commenting them out till used?